File: GoLang/fizzbuzz.go
package main

import ( 
	"fmt" 
	"flag"
)

var max *int = flag.Int("max", 0, "enter integer or bust!") 

func main() {
	flag.Parse()
	for i := 1; i <= *max; i++ {
		fizzbuzz(i)	
	}
}

func fizzbuzz(i int) {
	fizz := "fizz"
	buzz := "buzz"
	
	if i % 3 == 0 && i % 5 == 0 {
		fmt.Println(i, fizz + buzz)
	} else if i % 3 == 0 {
		fmt.Println(i, fizz)
	} else if i % 5 == 0 {
		fmt.Println(i, buzz)
	} else {
		fmt.Println(i)
	}
}

File: Add Code Here/PYTHON/Basic Pyhton Programs/fizzbuzz.py
def fizzbuzz(num: int) -> list: 
  
     buzz_list = [] 
  
     for i in range(1, num): 
         if i % 5 == 0 and i % 3 == 0: 
             buzz_list.append("fizzbuzz") 
         elif i % 5 == 0: 
             buzz_list.append("buzz") 
         elif i % 3 == 0: 
             buzz_list.append("fizz") 
         else: 
             buzz_list.append(i) 
  
     return buzz_list
